### PR TITLE
Update README and logo, provide a build script

### PR DESCRIPTION
**Pull request by slvrtrn**
* README now contains updated documentation regarding a development environment setup, building from source, etc
* We have to replace the awesome Metabase logo in ClickHouse colours with a plain simple ClickHouse one cause the current one does not conform to Metabase branding guidelines.

### CODE REVIEW

**Review by enqueue — COMMENTED** [review]
Can we include the Metabase logo "as is"?

**Comment by slvrtrn** [comment]
> Can we include the Metabase logo "as is"?

I asked to double-check

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
```
[the remark this answers is not in the document]
Comment (slvrtrn) [reply build.sh:2]: Yeah, this is a good one

**Comment by slvrtrn** [comment]
@enqueue confirmed - the original Metabase logo is OK. Please let me know what you think of the current version.

**Review by enqueue — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,25 +1,59 @@
...
-2. Download the ClickHouse driver jar from this repository's "Releases" page
+2. Download the ClickHouse driver jar from this repository's [Releases](https://github.com/enqueue/metabase-clickhouse-driver/releases) page
 3. Create a directory and copy the `metabase.jar` to it.
 4. In that directory create a sub-directory called `plugins`.
 5. Copy the ClickHouse driver jar to the `plugins` directory.
 6. Make sure you are the in the directory where your `metabase.jar` lives.
 7. Run `MB_PLUGINS_DIR=./plugins; java -jar metabase.jar`.
 
-### Choosing the Right Version
+For example (using Metabase v0.44.6 and ClickHouse driver 0.8.3):
```
Comment (mshustov) [inline README.md:35]: let's add a link to `Choosing the Right Version` section

File: README.md [hunk 3]
```diff
@@ -33,52 +67,143 @@ Metabase Release | Driver Version
 0.42.x           | 0.8.1
 0.44.x           | 0.8.3
 
-### Building from Source
+## Setting up a development environment
```
Comment (mshustov) [inline README.md:70]: This content is more appropriate for CONTRIBUTING.md file

File: README.md [hunk 4]
```diff
@@ -1,25 +1,59 @@
...
-
-[ClickHouse](https://clickhouse.yandex) ([github](https://github.com/ClickHouse/ClickHouse)) database driver for the [Metabase](https://metabase.com) ([github](https://github.com/metabase/metabase)) business intelligence front-end
-
-![Metabase Logo in ClickHouse colours](docs/images/mbch_logo.png)
-
-[![Latest Release](https://img.shields.io/github/release/enqueue/metabase-clickhouse-driver.svg?label=latest%20release)](https://github.com/enqueue/metabase-clickhouse-driver/releases)
-[![GitHub license](https://img.shields.io/badge/license-AGPL-05B8CC.svg)](https://raw.githubusercontent.com/enqueue/metabase-clickhouse-driver/master/LICENSE.txt)
+<p align="center" style="font-size:300%">
+<img src="https://www.metabase.com/images/logo.svg" width="200px" align="center">
+<img src=".static/logo.png" width="200px" align="center">
```
[the remark this answers is not in the document]
Comment (slvrtrn) [reply README.md:3]: nice

**Pull request merged**